apps/api/src/app/events/usecases/send-message/send-message-type.usecase.ts

```diff
@@ -13,12 +14,22 @@ export abstract class SendMessageType {
     message,
     status: 'error' | 'sent' | 'warning',
     errorId: string,
-    errorMessage: string,
+    errorMessageFallback: string | any,
```

Comment:
```suggestion
    errorMessageFallback: string,
```
I think we can safely leave the type only as string as this param is being filled either with the error message or error name that are strings or a fallback string when it is being used.